src/Installers/Windows/AspNetCoreModule-Setup/IIS-Common/README.md

```diff
@@ -0,0 +1,18 @@
+Microsoft IIS Common
+--------------------------------
+
+The repository contains common resources shared by IIS Out-Of-Band (OOB) products.
```

Comment:
It's probably worth changing this to say "This code was copied from https://github.com/Microsoft/IIS.Common/tree/7d690a2135dfdac884a82b340f7114ecf27fc285"